client/src/components/dashboard/TemplateSelector.js

```diff
@@ -88,30 +87,33 @@ const TemplateSelector = (props) => {
             <Form.Check
               inline
               type="radio"
-              name="theme"
-              value="theme1"
-              onChange={updateForm}
+              onChange={() => updateSelections('theme', 1)}
               checked={theme === 1}
             />
-            <div className="theme-icon theme-1-icon" />
+            <div 
+              className="theme-icon theme-1-icon" 
+              onClick={() => updateSelections('theme', 1)} 
+            />
             <Form.Check
               inline
               type="radio"
-              name="theme"
-              value="theme2"
-              onChange={updateForm}
+              onChange={() => updateSelections('theme', 2)}
               checked={theme === 2}
             />
-            <div className="theme-icon theme-2-icon" />
+            <div 
+              className="theme-icon theme-2-icon" 
+              onClick={() => updateSelections('theme', 2)} 
+              />
             <Form.Check
               inline
               type="radio"
-              name="theme"
-              value="theme3"
-              onChange={updateForm}
+              onChange={() => updateSelections('theme', 3)}
               checked={theme === 3}
             />
-            <div className="theme-icon theme-3-icon" />
+            <div 
+              className="theme-icon theme-3-icon" 
+              onClick={() => updateSelections('theme', 3)} 
+              />
```

Comment:
Just a suggestion for improvement is it possible to combine the `<Form.Check> `and `<div>` into a single div so we only handle the onClick function in a single place instead of 2?

Reply:
@ssha0054  Good news, I found a way to wrap them, see [here](https://stackoverflow.com/questions/19412204/how-do-you-make-the-radio-button-text-to-be-clickable-too/54573527)